x-pack/metricbeat/module/aws/cloudwatch/cloudwatch_test.go

```diff
@@ -1306,3 +1309,59 @@ func TestCreateEventsWithoutIdentifier(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, value2, dimension)
 }
+
+func TestInsertTags(t *testing.T) {
+	identifier1 := "StandardStorage,test-s3-1"
+	identifier2 := "test-s3-2"
+	tagKey1 := "organization"
+	tagValue1 := "engineering"
+	tagKey2 := "owner"
+	tagValue2 := "foo"
+
+	events := map[string]mb.Event{}
+	events[identifier1] = aws.InitEvent(regionName, accountName, accountID)
+	events[identifier2] = aws.InitEvent(regionName, accountName, accountID)
+
+	resourceTagMap := map[string][]resourcegroupstaggingapi.Tag{}
+	resourceTagMap["test-s3-1"] = []resourcegroupstaggingapi.Tag{
+		{
+			Key:   awssdk.String(tagKey1),
+			Value: awssdk.String(tagValue1),
+		},
+	}
+	resourceTagMap["test-s3-2"] = []resourcegroupstaggingapi.Tag{
+		{
+			Key:   awssdk.String(tagKey2),
+			Value: awssdk.String(tagValue2),
+		},
+	}
+
+	cases := []struct {
+		title            string
+		identifier       string
+		expectedTagKey   string
+		expectedTagValue string
+	}{
+		{
+			"test identifier with only s3 bucket name",
```

Comment:
Shouldn't this use `identifier2`?

Reply:
hmmm sorry I probably didn't get what you mean here. `identifier2` is in the next test case in Ln 1353.

Reply:
sorry I finally realized whats wrong here. The title of the two test cases are flipped... Thanks for catching this. Fixed now!